File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/RaidOption.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * A RAID configuration option available for a dedicated server.
 */
class RaidOption
{
    #[DTA\Data(field: "id", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "int"])]
    public int|null $id = null;

    #[DTA\Data(field: "short_desc", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $short_desc = null;

    #[DTA\Data(field: "monthly_price", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "float"])]
    public float|null $monthly_price = null;

}

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/RaidOption.scala
package io.swagger.server.model

import java.math.BigDecimal

/**
 * A RAID configuration option available for a dedicated server.
 *
 * @param id 
 * @param short_desc 
 * @param monthly_price 
 */
case class RaidOption (
  id: Option[Int],
  short_desc: Option[String],
  monthly_price: Option[BigDecimal]
)

